core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.ts

```diff
@@ -254,10 +254,13 @@ export class CustomizeInteractionModalComponent
   }
 
   isSaveInteractionButtonEnabled(): boolean {
+    let exploration = this.getTitle(this.stateInteractionIdService.displayed);
```

Comment:
```suggestion
    let explorationTitle = this.getTitle(this.stateInteractionIdService.displayed);
```

Reply:
this change will fail the lint check because the line will exceed the limit of 80 characters

Reply:
You can break the line from the parens like so:
```javascript
let explorationTitle = this.getTitle(
  this.stateInteractionIdService.displayed);
```